README.md

```diff
@@ -459,7 +459,26 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 - `ciphers` is list of enabled ciphers. The values are not checked.
 
 Please look into `examples` folder that contains `olsconfig.yaml` with filled-in TLS security profile for the service.
-Additionally the TLS security profile can be set for any configured provider. In this case the `tlsSecurityProfile` needs to be added into the `olsconfig.yaml` file into `llm_provieders/{selected_provider}` section.
+Additionally the TLS security profile can be set for any configured provider. In this case the `tlsSecurityProfile` needs to be added into the `olsconfig.yaml` file into `llm_provieders/{selected_provider}` section. For example:
```

Comment:
I know you didn't change this part but llm_providers*

Reply:
fixed, thank you